WaterLog_Backend/Controllers/SegmentLeaksController.cs

```diff
@@ -86,6 +86,16 @@ public async Task<ActionResult<SegmentLeaksEntry>> Get(int id)
             return leaks;
         }
 
+
```

Comment:
Remove double whitespace, should only be single between methods

Reply:
Fixed